roles/openshift_aws/defaults/main.yml

```diff
@@ -46,6 +47,45 @@ openshift_aws_ami_tags:
 openshift_aws_s3_mode: create
 openshift_aws_s3_bucket_name: "{{ openshift_aws_clusterid }}-docker-registry"
 
+openshift_aws_dns_dict:
+  external:
+    provider: ''  # [ dyn, route53 ]
+    #customer_name: 'customer_name'  # Reqd for dyn
+    #user_name: 'user_name'  # Reqd for dyn
+    #user_passwd: 'user_passwd'  # Reqd for dyn
+    zone: "{{ openshift_aws_clusterid }}.example.com"
+    records:
```

Comment:
break out the records to their own list of dictionaries called openshift_aws_dns_records